recipes/example/meta.yaml

```diff
@@ -79,4 +79,3 @@ extra:
     # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
     - LisaSimpson
     - LandoCalrissian
-
```

Comment:
There is still an edit to this file. This should stay as-is

Reply:
Yup, sorry about that. Fixed.